src/main/java/com/zendesk/maxwell/schema/columndef/ColumnDefTest.java

```diff
@@ -0,0 +1,358 @@
+package com.zendesk.maxwell.schema.columndef;
```

Comment:
`git mv` is just a delete and an add -- other than that there's just an algorithm to detect renames.  If you change too much of the file, you confuse the rename algorithm.  A pity, really.